lib/screens/home.dart

```diff
@@ -0,0 +1,35 @@
+
+import 'package:flutter/material.dart';
+
+import '../models/listitem.dart';
+
+class MyApp extends StatelessWidget {
+  final List<ListItem> items;
+
+  const MyApp({Key? key, required this.items}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    const title = 'Mixed List';
+
+    return MaterialApp(
+      title: title,
+      home: Scaffold(
+        appBar: AppBar(
+          title: const Text(title),
+        ),
+        body: ListView.builder(
+          itemCount: items.length,
+          itemBuilder: (context, index) {
```

Comment:
I can even highlight multiple lines

Reply:
Does everyone has permission to approve?